test/Datadog.Trace.TestHelpers/Datadog.Trace.TestHelpers.csproj

```diff
@@ -1,5 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Instead of overriding the default target frameworks here and targetting `net45` in `Datadog.Core.Tools.csproj`, can we try targeting `net461` in the executable tool? The integration that are limited to netframework only should be available for either target, and this seems like a cleaner fix (if it works).